### PR TITLE
feat: adds an optional check to see if issuer property matches

### PR DESCRIPTION
**Pull request by kdenhartog**
This addresses issue #18

### CODE REVIEW

**Comment by kdenhartog** [comment]
I used VSCode's Javascript formatter, but it looks like it updated more than the code that I added. Do you guys normally use just 80 char text wrapping for formatting or some other tool?

**Comment by LoveishSarolia** [comment]
> I used VSCode's Javascript formatter, but it looks like it updated more than the code that I added. Do you guys normally use just 80 char text wrapping for formatting or some other tool?

If this answers your question, we use the ESLint extension in VSCode for base-level formatting.

**Comment by kdenhartog** [comment]
> > I used VSCode's Javascript formatter, but it looks like it updated more than the code that I added. Do you guys normally use just 80 char text wrapping for formatting or some other tool?
> 
> If this answers your question, we use the ESLint extension in VSCode for base-level formatting.

Cleaned it up now following that format. Thanks for the tip.

**Comment by mattcollier** [comment]
@kdenhartog thank you for the PR, we'll get some eyes on this soon, but it may take several days.

**Review by dmitrizagidulin — APPROVED** [review]
👍  from me, thanks @kdenhartog

File: main.js [hunk 1]
```diff
@@ -181,6 +193,26 @@ async function _checkStatus({
     }
   }
 
+  /**
+   * This checks that the issuer of the verifiable credential matches
+   * the issuer of the revocationListCredential
+   */
```
Comment (dlongley) [inline main.js]: We use this comment style for simple comments on conditionals, short code blocks, etc:

```suggestion
  // this checks that the issuer of the verifiable credential matches
  // the issuer of the revocationListCredential
```

The other style is reserved for jsdocs/descriptions of classes, functions, etc.

**Review by dlongley — APPROVED** [review]
LGTM -- can be merged after minor style fix on a comment.

**Comment by dlongley** [comment]
Thanks, @kdenhartog!

**Comment by davidlehn** [comment]
Our workflow setups are not running on external PRs.  Whoever plans to merge, please run test and lint scripts by hand first.

**Comment by kdenhartog** [comment]
rebased the latest js style commit on and fixed the comment. Should be all ready to go now.

**Comment by kdenhartog** [comment]
Any update needed here? @dmitrizagidulin @dlongley @mattcollier @gannan08

**Comment by mattcollier** [comment]
Work on this continues here: https://github.com/digitalbazaar/vc-revocation-list/pull/23

**Pull request merged**